modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -318,6 +319,10 @@ public CompletableFuture<Void> execute(CatalogCommand command) {
 
     @Override
     public CompletableFuture<Void> execute(List<CatalogCommand> commands) throws IllegalArgumentException {
```

Comment:
seems IllegalArgumentException can be thrown here, let's remove it

Reply:
removed